hbase-server/src/test/java/org/apache/hadoop/hbase/replication/regionserver/TestMetaRegionReplicaReplicationEndpoint.java

```diff
@@ -63,17 +68,20 @@
 /**
  * Tests RegionReplicaReplicationEndpoint class for hbase:meta by setting up region replicas and
  * verifying async wal replication replays the edits to the secondary region in various scenarios.
+ *
  * @see TestRegionReplicaReplicationEndpoint
  */
 @Category({LargeTests.class})
```

Comment:
Not yours, but I think this can be be added to the `MasterTests` category. Or `RegionServerTests`.. I'm not even clear on which one this is at this point. Maybe need a category for `CatalogueTests`.

Reply:
will add "RegionServerTests" tag.